docs/application/native/guides/compliations/watchface-complication.md

```diff
@@ -317,21 +309,20 @@ static void _watchface_editable_update_requested_cb(const watchface_editable_h h
 ```
 
 
-# Transfers Touch Event
+## Transferring Touch Event
 
 Complication can provide additional action for touching when the complication is touched.
-To trigger the complication touch action,  watchface should transfer the touch event to complication provider when the
-complication is touched.
+To trigger the complication touch action,  watchface must transfer the touch event to complication provider when the complication is touched.

```

Comment:
To trigger the complication touch action, watchface must transfer the touch event to complication provider when the complication is touched.

Please remove additional space between , and "watchface"